cmd/okctl/create.go

```diff
@@ -47,6 +47,12 @@ func buildCreateCommand(o *okctl.Okctl) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "create",
 		Short: "Create commands",
+		Long: `Create various kinds of resources.
+
+Pre-requisites:
+* okctl creates various configuration files, and assumes that it is
```

Comment:
Remove the '*' in the start of this sentence (?)

Reply:
I was thinking about this point as a list. Although there's just element now, so I can remove the `*` and possibly re-add it if we ever get one more requirement.